### PR TITLE
Removing Cypress

### PR DESCRIPTION
**Pull request by Mitch-Budibase**
We no longer need Cypress.

I am removing the Cypress directory and making any other necessary changes

### CODE REVIEW

**Review by Pedro-Revez-Silva — APPROVED** [review]
LGTM

**Review by MihailHadzhiev2022 — APPROVED** [review]

File: packages/builder/setup.js [hunk 1]
```diff
@@ -1,14 +1,14 @@
-const cypressConfig = require("../cypress.json")
```
Comment (shogunpurple) [inline packages/builder/setup.js:1]: I would possibly rename this file as `testSetup.js` or similar as it's only for testing.

File: packages/builder/ts/setup.ts [hunk 2]
```diff
@@ -0,0 +1,4 @@
+// @ts-ignore
```
Comment (shogunpurple) [inline packages/builder/ts/setup.ts:1]: Same as above.

**Review by shogunpurple — APPROVED** [review]
Code LGTM - just a minor naming comment and looks like the qa-core tests are failing due to this change. Once fixed it's good to go!

**Comment by aptkingston** [comment]
The day has finally come :pray: 

I'll hope on this branch and remove all traces of `data-cy` in our components. Makes sense to include that removal in this PR too.

File: packages/builder/ts/setup.ts (hunk 2 above)
Comment (Rory-Powell) [reply packages/builder/ts/setup.ts:1]: We could probably move this out of the `builder` now all together, as it's only api related. It should probably live it qa-core itself

File: packages/builder/setup.js (hunk 1 above)
Comment (Rory-Powell) [reply packages/builder/setup.js:1]: Similar to other comment, we can move this directly into qa-core now

**Pull request merged**